language-server/modules/langserver-core/src/test/resources/completion/statement_context/source/assignment_stmt_ctx_source12.bal

```diff
@@ -0,0 +1,4 @@
+function testFunction() {
+    string testString = "testString";
```

Comment:
Is there a particular reason why this `testString` was added? If not, can we proceed to remove it? Make the test case unnecessarily complex IMO.

Reply:
No specific reason. +1 to remove the line.